packages/env/test/defaults.test.js

```diff
@@ -171,4 +171,20 @@ describe('Defaults', () => {
 
     expect(env).toHaveProperty('git.sha', null);
   });
+
+  describe('env PERCY_SKIP_GIT_CHECK', () => {
+    beforeEach(() => {
+      process.env.PERCY_SKIP_GIT_CHECK = true;
+    });
+
+    afterAll(() => {
+      process.env.PERCY_SKIP_GIT_CHECK = false;
+    });
+
+    it('does not make git commands, if there are any', () => {
```

Comment:
Don't we always execute git ? Is there a condition for 'if there are any' ?

Reply:
updated the description